scripts/ubuntu-bartender/ubuntu-bartender

```diff
@@ -262,6 +269,12 @@ build-provider-create $bartender_name
     cp -aR "$LIVECD_ROOTFS_DIR" $temp_dir/livecd-rootfs
   fi
 
+  if [ -n "$CHROOT_ARCHIVE" ]
+  then
+    cp -aR "$CHROOT_ARCHIVE" $temp_dir/chroot-archive
+    chroot_archive_flag="--use-chroot-archive ../chroot-archive"
```

Comment:
Do we want to explicitly set $chroot_archive_flag to "" if this if branch is not taken?

Reply:
I don't think so. By default the variable will be unset/empty.